Source/Foundation/bsfCore/BsCoreApplication.cpp

```diff
@@ -75,7 +75,7 @@ namespace bs
 		// Ensure all errors are reported properly
 		CrashHandler::startUp(desc.crashHandling);
 		if(desc.logCallback)
-			gDebug().getLog().setLogCallback(desc.logCallback);
+			gDebug().setLogCallback(desc.logCallback);
```

Comment:
Perhaps it would be better to make this an `Event`? This way multiple things can subscribe to it.

Reply:
Ah but I guess you need the return value. So that's not going to work.

Reply:
Yeah, I want to skip the default thing. And it looks like the Event class can't return any callback result values.